Vagrantfile

```diff
@@ -114,4 +111,11 @@ Vagrant.configure(2) do |config|
     configure_streamalert(py3)
     final_message(py3)
   end
+
+  config.ssh.forward_env = [
+    'AWS_DEFAULT_REGION',
+    'AWS_ACCESS_KEY_ID',
```

Comment:
Is it possible forward `AWS_PROFILE`? I use `AWS_PROFILE` so I don't export my `AWS_ACCESS_KEY_ID ` and `AWS_SECRET_ACCESS_KEY ` to the shell env. But the real credential is saved in ~/.aws/ directory. So I'm not clear is `AWS_PROFILE` forwarding working in the VM.

Reply:
to forward `AWS_PROFILE` we would need to mount your `~/.aws/` into the VM, which i would prefer not to do. I expose my credentials by using [aws-vault](https://github.com/99designs/aws-vault). i then can do things like `aws-vault exec NAME -- vagrant ssh` which will set my env_vars accordingly and support assuming roles